pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -729,20 +727,18 @@ func parseOneSystemAdapterConfig(getconfigCtx *getconfigContext,
 		networkXObject, err := getconfigCtx.pubNetworkXObjectConfig.Get(sysAdapter.NetworkUUID)
 		if err != nil {
 			// XXX when do we retry looking for the networkXObject?
-			errStr := fmt.Sprintf("parseSystemAdapterConfig: Port %s Network with UUID %s not found: %s",
+			err := fmt.Errorf("parseSystemAdapterConfigr Port %s Network with UUID %s not found: %s",
 				port.IfName, sysAdapter.NetworkUUID, err)
-			log.Error(errStr)
-			port.ParseError = errStr
-			port.ParseErrorTime = time.Now()
+			log.Error(err)
+			port.SetError(err, "zedagent parseconfig")
```

Comment:
These errors will be user-visible, hence should be actionally. I'd argue that the old message string "parseSystemAdapterConfig:"  doesn't help explain the issue and how the user should resolve it. But adding a "zedagent parseconfig" to the string makes it worse.

Reply:
same as above.. fixed